### ISSUE TITLE
Dimensions of Embeddings

### ISSUE THREAD

**Issue by stech-apps** [timesler/facenet-pytorch#102]
I'm using the inference code on fine-tuned model to get the face embeddings which I then want to save in vector index for similarity search:

    resnet = InceptionResnetV1(classify=True, num_classes=6) # where model was trained on 6 person's faces
    resnet.load_state_dict(torch.load(trained_model))
    resnet.eval().to(device)

and then calculating the embeddings:

    embeddings = resnet(aligned).detach().cpu()

but the embeddings I'm getting are not 512-dimensional?

**Comment by timesler** [comment]
You need to set `classify=False` to have the model return the embeddings.
